ui/pages/onboarding-flow/import-srp/import-srp.js

```diff
@@ -68,45 +48,20 @@ export default function ImportSRP({ submitSecretRecoveryPhrase }) {
       </div>
       <div className="import-srp__actions">
         <Box textAlign={TEXT_ALIGN.LEFT}>
-          <Typography variant={TYPOGRAPHY.H4}>
-            {t('secretRecoveryPhrase')}
-          </Typography>
-
-          <div className="srp-text-area">
-            <button onClick={() => setRevealSRP(!revealSRP)}>
-              <i
-                className={`far fa-eye${revealSRP ? '-slash' : ''}`}
-                style={{ color: 'var(--color-icon-default' }}
-              />
-            </button>
-            <textarea
-              data-testid="import-srp-text"
-              className={classnames('srp-text-area__textarea', {
-                'srp-text-area__textarea--blur': !revealSRP,
-                'srp-text-area__textarea--error': error,
-              })}
-              onChange={({ target: { value } }) =>
-                handleSecretRecoveryPhraseChange(value)
-              }
-              onPaste={clearClipboard}
-              autoComplete="off"
-              autoCorrect="off"
-            />
-            {error && (
-              <span className="srp-text-area__textarea__error-message">
-                {error}
-              </span>
-            )}
-          </div>
+          <SrpInput
+            onChange={setSecretRecoveryPhrase}
+            srpText={t('typeYourSRP')}
+          />
           <Button
             type="primary"
             data-testid="import-srp-confirm"
             large
+            className="import-srp__confirm-button"
             onClick={() => {
               submitSecretRecoveryPhrase(secretRecoveryPhrase);
               history.replace(ONBOARDING_CREATE_PASSWORD_ROUTE);
             }}
-            disabled={error || secretRecoveryPhrase.length === 0}
+            disabled={!secretRecoveryPhrase.trim()}
```

Comment:
is the `trim()` necessary? This value should always be either an empty string or a valid SRP. It will never be just whitespace.

```suggestion
            disabled={secretRecoveryPhrase === ''}
```